packages/array-max/src/index.js

```diff
@@ -3,9 +3,13 @@ import isArray from 'is-any-array';
 /**
  * Computes the maximum of the given values
  * @param {Array<number>} input
+ * @param {int} [options.from] - Start index (inclusive) for the slice within which we look for the max
+ * @param {int} [options.to] - End index (exclusive) for the slice within which we look for the max
  * @return {number}
  */
-export default function max(input) {
+export default function max(input, options = {}) {
+  let { from, to } = options;
```

Comment:
```suggestion
  let { from = 0, to = input.length } = options;
```

- Move this line after the validation of `input` to avoid errors if that is not an array
- You can then remove the "if undefined" conditions below